pipelines/prefect_flows/common_stages.py

```diff
@@ -68,6 +69,10 @@ def load_data(
         dist_alerts, method="nearest", tolerance=1e-5
     )
     subregion_aligned = xr.align(dist_alerts, subregion, join="left")[1]
+    pixel_area = _load_zarr(pixel_area_uri).reindex_like(
+        dist_alerts, method="nearest", tolerance=1e-5
+    )
+    pixel_area_aligned = xr.align(dist_alerts, pixel_area, join="left")[1]
```

Comment:
unrelated question for @solomon-negusse since I may have messed this up elsewhere, do you still need to do `align` even after `reindex_like`? Is the `reindex_like` just the snap, and the `align` is the clip?

Reply:
Exactly @jterry64, the groupby operation expects all layers to be lined up.